shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -1117,8 +1191,28 @@ public boolean shouldRestoreAndSaveState() {
     return true;
   }
 
+  /**
+   * {@inheritDoc}
+   *
+   * <p>Avoid overriding this method when using {@code
+   * shouldAutomaticallyHandleOnBackPressed(true)}. If you do, you must always {@code return
+   * super.popSystemNavigator()} rather than {@code return false}. Otherwise the navigation behavior
```

Comment:
This might make one think what if I return true :)? Anything we can say to save them a trial?

Reply:
Returning true is unproblematic, which is why I omitted it.